docker/ubuntu2004/base/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+FROM ubuntu:20.04 as carbon-ubuntu2004-base
```

Comment:
My leaning would be to go to 22.04 because it's latest at https://hub.docker.com/_/ubuntu, is there a particular reason for 20.04?

Reply:
Upgraded to 22.04